pkg/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -35,22 +33,27 @@ type RegistryConfig struct {
 	InsecureRegistryCIDRs []string
 }
 
-func getRegistryConfig(env *docker.Env) (*RegistryConfig, error) {
-	for _, entry := range *env {
-		if !strings.HasPrefix(entry, "RegistryConfig=") {
-			continue
+func getRegistryConfig(env *docker.DockerInfo) (*RegistryConfig, error) {
+	// TODO: Disabled because the latest go-dockerclient dropped docker.Env and
```

Comment:
I'll look into updating it. Ideally I should switch the code to use the new client.